autometa/common/external/diamond.py

```diff
@@ -29,146 +29,301 @@
 import os
 import subprocess
 
+import multiprocessing as mp
+
 from itertools import chain
 from tqdm import tqdm
 
-from autometa.common.utilities import make_pickle,file_length
+from autometa.common.utilities import make_pickle, file_length
 from autometa.common.external import prodigal
+from autometa.taxonomy.ncbi import NCBI
 
 logger = logging.getLogger(__name__)
 
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+NCBI_DIR = os.path.join(BASE_DIR, "databases", "ncbi")
```

Comment:
I'm not sure this is necessary.

Reply:
I added these to used `self.merged` here:
https://github.com/Sidduppal/Autometa/blob/72fc07b3dd0d3503f71da7447b04aa653b0c85e6/autometa/common/external/diamond.py#L551-L552

But after this [comment](https://github.com/KwanLab/Autometa/pull/87#discussion_r436074791), it looks like my solution was not correct after all. I will re-think how to search the `merge.dmp`. any suggestions?